cuda/private/cuda_helper.bzl

```diff
@@ -204,18 +204,21 @@ def _create_common(ctx):
     """
     attr = ctx.attr
 
+    all_ccdeps = [dep for dep in attr.deps if CcInfo in dep]
+    if hasattr(attr, "_builtin_deps"):
+        all_ccdeps.extend([dep for dep in attr._builtin_deps if CcInfo in dep])
+
+    cc_info = cc_common.merge_cc_infos(cc_infos = [dep[CcInfo] for dep in all_ccdeps])
```

Comment:
nit: I think `all_ccdeps` and `cc_info` better to be renamed to `all_cc_deps` and `merged_cc_info` respectively.

Reply:
Done